api/src/pcapi/sandboxes/scripts/creators/test_cases/__init__.py

```diff
@@ -11,6 +11,101 @@
 
 def save_test_cases_sandbox() -> None:
     create_offers_with_gtls()
+    create_offers_with_same_ean()
+
+
+venues = [
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏Chaud pour mettre ça dans un fichier à part
C'est un peu lourd de tous laisser dans le `__init__`